torch/onnx/symbolic_opset9.py

```diff
@@ -1409,12 +1409,20 @@ def __not_(g, self):
 
 
 def eq(g, self, other):
+    if isinstance(self.type(), DeviceObjType) and isinstance(
+        other.type(), DeviceObjType
+    ):
+        # ONNX doesn't have devices, so replace all of them with the same arbitrary
+        # tensor. The no-op check for equality will get constant-folded.
+        value_t = torch.tensor(1.0)
```

Comment:
why not directly return Constant of single bool value True?

Reply:
Done. Thanks for the suggestion.